src/Microsoft.OData.Core/Json/ODataUtf8JsonWriter.Stream.cs

```diff
@@ -0,0 +1,295 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataUtf8JsonWriter.Stream.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+#if NETCOREAPP
+using System;

```

Comment:
nit: In this project, the style seems to have imports nested within the namespace block